### PR TITLE
Onishchenko d task1

### PR DESCRIPTION
**Pull request by Dashawvr**
I did my homework1, please take a look.

### CODE REVIEW

File: src/components/app/App.js [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
 
-      {/*  todo: здесь нужно использовать UserCard по аналогии с Header, как пропсу user передать usersList[0]*/}
-    </div>
-  );
+function App() {
+    return (
+        <div className="App">
+            <Header />
+            <div className='d-flex'>
+                {usersList.map((user)=> <UserCard user={user}/>) }
```
Comment (AlyonaPianykh) [inline src/components/app/App.js]: nice! don't forget about `key` props

File: src/components/user-card/UserCard.js [hunk 2]
```diff
@@ -1,12 +1,33 @@
...
+            <div className='card-body'>
+                <h4 className='card-title'> {first_name} {last_name} </h4>
+                <div className='card-text'>
+                    <div>{email}</div>
+                    <div>{address}</div>
+                </div>
+            </div>
+        </div>
+    );
+
```
Comment (AlyonaPianykh) [inline src/components/user-card/UserCard.js:22]: This empty line can be deleted

**Comment by Dashawvr** [comment]
Ok, done!

**Comment by AlyonaPianykh** [comment]
Review done

**Pull request closed without merging**